framework/packages/abstract-std/src/native/ibc_host.rs

```diff
@@ -113,12 +113,12 @@ pub enum ExecuteMsg {
     /// Register the Polytone proxy for a specific chain.
     /// proxy should be a local address (will be validated)
     RegisterChainProxy {
-        chain: String,
+        chain: ChainName,
```

Comment:
Also rename these. 

We should either have 

`chain_name: ChainName`
or 
`destination: ChainName`
depending on the context. 

In the future we'll swap out `ChainName` with `ChainId`